src/borg/archive.py

```diff
@@ -80,13 +83,19 @@ def __add__(self, other):
         stats.osize_parts = self.osize_parts + other.osize_parts
         stats.usize_parts = self.usize_parts + other.usize_parts
         stats.nfiles_parts = self.nfiles_parts + other.nfiles_parts
+        stats.files_stats = other.files_stats
+        stats.chunking_time = other.chunking_time
+        stats.hashing_time = other.hashing_time
```

Comment:
I realized it, yes. They are needed so `archive.stats += fso.stats` (create_cmd.py line 188) moves those variables from object to another.
If this is done, it is not needed to modify the calls to `json_print` and `log_multi`.

Reply:
What I meant is that you are implementing an **add** operation here, so just assigning the stuff is not the correct operation. Even if the only use case was adding it to zero base values, you still need to do the addition.

Reply:
Ok, I understand what you mean. There is nothing to add here, so I'll have to move those lines somewhere else.
They could be moved to `create_cmd.py` below line 188
```python
    archive.stats += fso.stats
    archive.stats.files_stats = fso.stats.files_stats
    archive.stats.chunking_time = fso.stats.chunking_time
    archive.stats.hashing_time = fso.stats.hashing_time
```
And then it will go on and print the statistics.

Reply:
If the values are initializes to 0 or 0.0 (as they should be), adding them should be no problem?